src/main/scala/ztis/twitter/TweetProcessorActor.scala

```diff
@@ -103,6 +105,12 @@ class TweetProcessorActor(tweet: Tweet,
 
         log.info(s"Persisting $toPersist")
         cassandraClient.updateRating(toPersist)
+        
+        userResponse.get.followedBy.foreach { fromUserID => 
+          val association = UserVideoImplicitAssociation(fromUserID, UserOrigin.Twitter, userID, UserOrigin.Twitter, videoID, videoOrigin)
+          cassandraClient.updateImplicitAssociation(association)
+        }
+        relationshipsFetcher.queueFetchRelationshipsFor(tweet.userId())
```

Comment:
Maybe the name of the method is misleading - the idea is that you queue the request for relationships fetching. I will rename the method.